### PR TITLE
Update mapbox sample code

### PR DESCRIPTION
**Pull request by JeesubKim**
I updated mapbox sample code which shows slide mode of mapbox.
And added mapboxKey value.

### CODE REVIEW

File: src/components/sticker/MapViewMapBox.js [hunk 1]
```diff
@@ -2,99 +2,101 @@ import React from 'react';
 import { MapBox } from '@stickyboard/mapbox';
 
 const sampleInputs = {
-    data: {
-        mapboxKey: MAPBOX_TOKEN,
-        title: 'Stickyboard-mapbox example',
+    data:{
+        mapboxKey: 'pk.eyJ1IjoiamVlc3ViIiwiYSI6ImNrZTAxemlnOTBmZmYycnA3aG8wcmo1eWkifQ.wTZ5zzP0m-nh8JrhcIu94A',
```
Comment (soaple) [inline src/components/sticker/MapViewMapBox.js]: Please remove your MapBox API key here.

**Review by soaple — CHANGES_REQUESTED** [review]
Please check the comments.

File: src/components/sticker/MapViewMapBox.js (hunk 1 above)
Comment (JeesubKim) [reply src/components/sticker/MapViewMapBox.js]: I removed API key. Please check it.

**Review by soaple — APPROVED** [review]
LGTM

**Pull request merged**